### PR TITLE
Challenge/alfredo zimeperz/book quiz

### PR DESCRIPTION
**Pull request by fredyzz**

### CODE REVIEW

**Review by IrenePerezAlonso — APPROVED** [review]
De diez!! :D

**Review by jasuarez21 — DISMISSED** [review]
noce parec falzo

File: alfredo-zimperz/challenges/to-do-redux/src/services/api.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+function fetchBooks(
+  urlParams = { baseURL: GOOGLE_BOOKS_BASE_URL, volumesURL: VOLUMES_URL },
+  apiKey = credentials.API_KEY,
+) {
+  const url = urlParams.baseURL + urlParams.volumesURL + apiKey;
+  return fetch(url)
+    .then((response) => response.json());
+}
+
```
Comment (gteixi) [inline alfredo-zimperz/challenges/to-do-redux/src/services/api.js:14]: buenísimo fredy!!!!!!

**Review by gteixi — APPROVED** [review]

File: alfredo-zimperz/challenges/to-do-redux/src/components/Dashboard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  const title = 'Dashboard';
+  return (
+    <div>
+      <h2>{title}</h2>
+      <ul>
+        {
+        (books) && (
+          books.map((book) => (
+            <li>
```
Comment (shd8) [inline alfredo-zimperz/challenges/to-do-redux/src/components/Dashboard/index.jsx:22]: You should add a key for rendering components in loop. In this example you could do:

`<li key={book.id} >`

**Review by shd8 — APPROVED** [review]
Good job, use the corrections to improve your future projects.

**Review by gteixi — APPROVED** [review]

**Review by gaxxrodri — APPROVED** [review]

**Review by annasm07 — APPROVED** [review]
de quien te has copiado?

File: alfredo-zimperz/challenges/to-do-redux/src/actions/actionCreators.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import actionTypes from './actionTypes';
+import apiServices from '../services/api';
+
+// export function loadBooks() {
+//   return apiServices.fetchBooks().then(({ items }) => ({
+//     type: actionTypes.LOAD_BOOKS,
+//     books: items,
+//   }));
+// }
+
```
Comment (cristian-bars) [inline alfredo-zimperz/challenges/to-do-redux/src/actions/actionCreators.js:10]: Eliminar codigo comentado!

**Pull request merged**